### PR TITLE
fix batteryStats not updating everywhere, better loading time

### PR DESCRIPTION
**Pull request by yashash1511**
removed old font-awesome4 (just 2 icons were used from the entire lib) and office-ui-fabric-core (4 icons were being used)
replaced with svg. this should result in slightly better loading times.

fixed batteryStatus not updating everywhere

added tailwind the right way.

### CODE REVIEW

File: src/reducers/settings.js [hunk 1]
```diff
@@ -1,82 +1,82 @@
...
   }
 
-  if(val=="togg"){
-    tmp[path[path.length-1]] = !tmp[path[path.length-1]]
-  }else{
-    tmp[path[path.length-1]] = val
+  if (val == "togg") {
+    tmp[path[path.length - 1]] = !tmp[path[path.length - 1]];
+  } else {
+    tmp[path[path.length - 1]] = val;
```
[the remark this answers is not in the document]
Comment (yashash1511) [reply src/reducers/settings.js:52]: this has been there before i didn't change it.

**Review by andrewstech — APPROVED** [review]
Apparently I have to review this. So good work.
egg-meme-face

**Comment by yashash1511** [comment]
image
do not merge there is a bug

File: src/reducers/settings.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Lukas-Batema) [reply src/reducers/settings.js:52]: That doesn't mean it doesn't need to be fixed :|

**Comment by Lukas-Batema** [comment]
@yashash1511 Please fix the Prototype pollution, after I revert this.

**Pull request merged**